zeebe/exporters/elasticsearch-exporter/src/main/java/io/camunda/zeebe/exporter/ElasticsearchExporterConfiguration.java

```diff
@@ -112,6 +112,7 @@ public boolean shouldIndexValueType(final ValueType valueType) {
       case CONDITIONAL_SUBSCRIPTION -> index.conditionalSubscription;
       case CONDITIONAL_EVALUATION -> index.conditionalEvaluation;
       case GLOBAL_LISTENER_BATCH -> index.globalListenerBatch;
+      case EXPRESSION -> index.expression;
```

Comment:
❓ Is there a reason why the value type expression may be exported? Meaning, the changes in the exporter would allow exporting the value type expression. But there is no template, etc. Must expressions be exported when using the elasticsearch-exporter?